assets/dev/scss/admin/_promotion.scss

```diff
@@ -0,0 +1,108 @@
+@mixin margin( $top: inherit, $right: inherit, $bottom: inherit, $left: inherit ) {
+	margin-block-start: $top;
+	margin-inline-end: $right;
+	margin-block-end: $bottom;
+	margin-inline-start: $left;
+}
+
+.e-feature-promotion {
+	--e-a-top-bar-height: 50px;
+	--e-a-content-area-spacing: 110px;
+
+	width: 1220px;
```

Comment:
Maybe?
Add `max-width: 100%`.

What happens if the screen is smaller?

Reply:
there is a media query for 1410px
So the width will never get to 1220px